app/javascript/app/components/footer/footer.js

```diff
@@ -119,11 +136,13 @@ const getLogos = pathname => {
       return [...basePartners];
     case '/ndcs':
     case '/countries':
-      return [...basePartners, ...ndcPartners];
-    case '/emission-pathways':
-      return [...basePartners, ...espPartners];
+      return [...basePartners, ...ndcPartners].sort(alphabetically);
+    case '/pathways/models':
```

Comment:
I think we should use just pathways. Models in this case is for the menu of the table. So it can be /scenarios or /indicators too and still will be the same page